### ISSUE TITLE
"Firefox can't establish a connection to the server" Error in Email Reports

### ISSUE THREAD

**Issue by klazaj** [apache/incubator-superset#9135]
I am running Superset with Docker. I am trying to set up the email report feature in Superset. I installed geckodriver inside the container
```
apt-get update && apt-get install -y firefox-esr

wget https://github.com/mozilla/geckodriver/releases/download/v0.24.0/geckodriver-v0.24.0-linux64.tar.gz
tar -x geckodriver -zf geckodriver-v0.24.0-linux64.tar.gz -O > /usr/bin/geckodriver
chmod +x /usr/bin/geckodriver
rm geckodriver-v0.24.0-linux64.tar.gz
```
 and configured the smtp in `config.py` like so:
```
# smtp server configuration
EMAIL_NOTIFICATIONS = False  # all the emails are sent using dryrun
SMTP_HOST = "smtp.gmail.com"
SMTP_STARTTLS = True
SMTP_SSL = False
SMTP_USER = "user@gmail.com"
SMTP_PORT = 587
SMTP_PASSWORD = "password"
SMTP_MAIL_FROM = "user@gmail.com"
WEBDRIVER_BASEURL = "http://0.0.0.0:8080/"
```

When I try to send a test email from Superset, I get the following error:
```
superset_1  | [2020-02-13 17:38:35,822: ERROR/MainProcess] Task email_reports.send[537487c3-a968-4439-b601-b22edcba8603] raised unexpected: WebDriverException('Reached error page: about:neterror?e=connectionFailure&u=http%3A//0.0.0.0%3A8080/superset/welcome&c=UTF-8&f=regular&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%200.0.0.0%3A8080.', None, None)
superset_1  | Traceback (most recent call last):
superset_1  |   File "/usr/local/lib/python3.6/site-packages/celery/app/trace.py", line 385, in trace_task
superset_1  |     R = retval = fun(*args, **kwargs)
superset_1  |   File "/usr/local/lib/python3.6/site-packages/celery/app/trace.py", line 648, in __protected_call__
superset_1  |     return self.run(*args, **kwargs)
superset_1  |   File "/home/superset/superset/tasks/schedules.py", line 365, in schedule_email_report
superset_1  |     deliver_dashboard(schedule)
superset_1  |   File "/home/superset/superset/tasks/schedules.py", line 210, in deliver_dashboard
superset_1  |     driver = create_webdriver()
superset_1  |   File "/home/superset/superset/tasks/schedules.py", line 169, in create_webdriver
superset_1  |     driver.get(welcome_url)
superset_1  |   File "/usr/local/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 333, in get
superset_1  |     self.execute(Command.GET, {'url': url})
superset_1  |   File "/usr/local/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
superset_1  |     self.error_handler.check_response(response)
superset_1  |   File "/usr/local/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
superset_1  |     raise exception_class(message, screen, stacktrace)
superset_1  | selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//0.0.0.0%3A8080/superset/welcome&c=UTF-8&f=regular&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%200.0.0.0%3A8080.
```
**TL;DR error code**: Firefox can't establish a connection to the server at localhost

What am I doing wrong here?  Why isn't the connection established?

**Comment by klazaj** [comment]
I had to change `WEBDRIVER_BASEURL = "http://0.0.0.0:8080/"` to `WEBDRIVER_BASEURL = "http://0.0.0.0:8088/"` I hadn't noticed that ._.
